tmt.spec

```
@@ -77,6 +77,8 @@ in a libvirt environment provisioned using testcloud.
 %package all
 Summary: Extra dependencies for the Test Management Tool
 Requires: tmt == %{version}-%{release}
+Requires: tmt-container == %{version}-%{release}
+Requires: tmt-testcloud == %{version}-%{release}
```

I'm not sure about the `==`... I've seen single `=` everywhere.

I also am not sure, why we wouldn't simply input version there? Even in case we have semi-stable api, it should never change on `release`.

So I'd actually put there something like

```suggestion
Requires: tmt-testcloud >= %{version}
```